apps/omg_watcher/lib/omg_watcher/db/eth_event.ex

```diff
@@ -43,7 +44,7 @@ defmodule OMG.Watcher.DB.EthEvent do
   """
   @spec insert_deposits!([OMG.State.Core.deposit()]) :: :ok
   def insert_deposits!(deposits) do
-    deposits |> Enum.each(&insert_deposit!/1)
+    Enum.each(deposits, &insert_deposit!/1)
```

Comment:
same here, not relevant (both chagnes to this file and there's other files that are changed detached from the PR at hand)